interpreter/cling/tools/packaging/cpt.py

```diff
@@ -1834,7 +1834,8 @@ def custom_input(prompt, always_yes=False):
 #                               Global variables                              #
 ###############################################################################
 
-workdir = os.path.abspath(os.path.expanduser(args['with_workdir']))
+if __name__ == "__main__":
```

Comment:
Can you clarify (in the commit message) what this change does and why it is needed?

Reply:
Hi @vgvassilev, I just made changes to the original commit message

Reply:
Can you format your commit message according to https://cbea.ms/git-commit/

Reply:
Yep just reformatted according to the link @vgvassilev , sorry for the wait

Reply:
I still see the old commit message style.

Reply:
ping.

Reply:
Sorry not getting the notifications for this PR for some reason

Reply:
Hi @vgvassilev, commit style fixed sorry for the wait